geode-client-native/src/cppcache/impl/CppCacheLibrary.cpp

```diff
@@ -79,7 +84,6 @@ typedef ACE_Singleton<CppCacheLibrary,ACE_Recursive_Thread_Mutex> TheLibrary;
 // internals are prepared. fw_helper framework will handle this.
 CppCacheLibrary* CppCacheLibrary::initLib( void )
 {
-  ACE::init();
```

Comment:
Why remove the init here? Is it moved somewhere else and are there some files in this change not listed in the PR/diff maybe?